rasa_scheduler/celery_tasks.py

```diff
@@ -1,5 +1,13 @@
+import os
+
 import requests
 from celery import Celery
+from dotenv import load_dotenv
+from virtual_coach_db.dbschema.models import Users
+from virtual_coach_db.helper.helper import get_db_session
+
+load_dotenv()
+DB_HOST = os.getenv('DB_HOST')
 
 app = Celery('celery_tasks', broker='redis://redis:6379')
```

Comment:
Do we also want to move the redis URL to some sort of central config location?

Reply:
For now, to help the developer the DB_HOST is in a .env file, because we run the database separately and the developer depending on the OS has to adapt the DB_HOST to for example: `host.docker.internal:5432`. I've created #148 for moving all hard-coded urls to a central config, which we will pickup in a later stage.